MetadataExtractor/Formats/Tga/TgaFooterReader.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Drew Noakes and contributors. All Rights Reserved. Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using MetadataExtractor.IO;
+using MetadataExtractor.Util;
+using System.IO;
+using System.Text;
+
+namespace MetadataExtractor.Formats.Tga
+{
+    struct TgaFooter
+    {
+        public int extOffset;
+        public int devOffset;
+        public byte[] signature;
+    }
+
+    /// <summary>Reads TGA image file footer.</summary>
+    /// <author>Dmitry Shechtman</author>
+    sealed class TgaFooterReader : TgaReader<TgaFooter, SequentialReader>
+    {
+        private const int FooterSize = 26;
+
+        private static readonly byte[] FooterMagic = Encoding.ASCII.GetBytes("TRUEVISION-XFILE.\0");
+
+        public static readonly TgaFooterReader Instance = new TgaFooterReader();
```

Comment:
Why did you make these singletons?

Reply:
These classes don't have states, but have virtual members, so they can't be made static.

Reply:
Why not new them up each time? There doesn't seem much benefit to keeping long-lived instances as far as I can tell.

Reply:
Done.